packages/target-nodejs-sdk/test/request.spec.js

```diff
@@ -110,43 +110,67 @@ describe("Requests to target delivery API", () => {
     expect(fetch.mock.calls.length).toEqual(1);
   });
 
-  describe("environment ID", () => {
```

Comment:
Did you mean to remove `environmentId` ?

Reply:
Yes, it was intentional, to be consistent with openapi-specs since `organizationId` is sufficient to identify a client without needing `environmentId`.